content/influxdb/cloud-serverless/get-started/query.md

```diff
@@ -212,11 +228,545 @@ WHERE
 
 5.  Click **{{< icon "play" >}} {{% caps %}}Run{{% /caps %}}**.
 
-Results are displayed under the text editor.
+Results are displayed under the query editor.
 
 See [Query in the Data Explorer](/influxdb/cloud-serverless/query-data/execute-queries/data-explorer/) to learn more.
 
-<!---------------------------- END FLUX UI CONTENT ---------------------------->
+<!---------------------------- END UI CONTENT ---------------------------->
+{{% /tab-content %}}
+{{% tab-content %}}
+<!--------------------------- BEGIN influx3 CONTENT --------------------------->
+
+Query InfluxDB v3 using SQL and the `influx3` CLI, part of the [`InfluxCommunity/pyinflux3`](https://github.com/InfluxCommunity/pyinflux3) community repository.
+The following steps include setting up a Python virtual environment already
+covered in [Get started writing data](/influxdb/cloud-serverless/get-started/write/?t=Python#write-line-protocol-to-influxdb).
+_If your project's virtual environment is already running, skip to step 3._
+
+1.  Setup your Python virtual environment.
+    Inside of your project directory:
+
+    ```sh
+    python -m venv envs/virtual-env
+    ```
+
+2. Activate the virtual environment.
+
+    ```sh
+    source ./envs/virtual-env/bin/activate
+    ```
+
+3. Install the following dependencies:
+
+    {{< req type="key" text="Already installed in the [Write data section](/influxdb/cloud-serverless/get-started/write/?t=Python#write-line-protocol-to-influxdb)" color="magenta" >}}
+
+    - `pyarrow` {{< req text="\*" color="magenta" >}}
+    - `flightsql-dbapi` {{< req text="\*" color="magenta" >}}
+    - `pyinflux3` {{< req text="\*" color="magenta" >}}
+
+4. Create the `config.json` configuration.
+
+    ```sh
+    influx3 config --name="my-config" --database="BUCKET_NAME" --host="cloud2.influxdata.com" --token="INFLUX_API_READ_TOKEN" --org="INFLUX_ORG_ID"
+    ```
+
+    Replace the following:
+
+    - **`BUCKET_NAME`**: the name of the InfluxDB Cloud Serverless bucket to query
+    - **`INFLUX_API_READ_TOKEN`**: InfluxDB API token with _read_ access to the **get-started** bucket
+    - **`INFLUX_ORG_ID`**: InfluxDB organization ID
+
+5. Enter the `influx3 sql` command and your SQL query statement.
+
+  ```sh
+  influx3 sql "SELECT * FROM home WHERE time >= '2022-01-01T08:00:00Z' AND time <= '2022-01-01T20:00:00Z'"
+  ```
+
```

Comment:
```suggestion

  {{% influxdb/custom-timestamps %}}
```sh
influx3 sql "SELECT * FROM home WHERE time >= '2022-01-01T08:00:00Z' AND time <= '2022-01-01T20:00:00Z'"
```
  {{% /influxdb/custom-timestamps %}}

```